modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/RowHandler.java

```diff
@@ -48,5 +51,7 @@ interface RowFactory<RowT> {
         RowT create();
 
         RowT create(Object... fields);
+
+        RowT create(ByteBuffer raw);
```

Comment:
I believe it's time to improve javadoc for this class. Please address at least newly introduced methods. It would be nice to cover few more methods